src/fb.c

```diff
@@ -214,6 +214,14 @@ void tfb_flush_window(void)
    tfb_flush_rect(0, 0, __fb_win_w, __fb_win_h);
 }
 
+void tfb_flush_fb(void)
+{ 
+   __fbi.activate |= FB_ACTIVATE_NOW | FB_ACTIVATE_FORCE;
+   if(0 > ioctl(fbfd, FBIOPUT_VSCREENINFO, &__fbi)) {
+      fprintf(stderr, "Failed to refresh\n");
```

Comment:
Please define something like:
```
#define TFB_ERR_FB_FLUSH_IOCTL_FAILED 16
```
In `tfb_errors.h` and append string version of the error in the `error_msgs` array in` errors.c`, instead of writing to stderr.

The string error could be something like: _Unable to flush the framebuffer with ioctl()_.